tests/snappi/bgp/files/bgp_convergence_helper.py

```diff
@@ -941,5 +941,5 @@ def cleanup_config(duthost):
     duthost.command("sudo cp {} {}".format("/etc/sonic/config_db_backup.json","/etc/sonic/config_db.json"))
     duthost.shell("sudo config reload -y \n")
     logger.info("Wait until all critical services are fully started")
-    pytest_assert(wait_until(360, 10, 1, duthost.critical_services_fully_started), "Not all critical services are fully started")
+    pytest_assert(wait_until(360, 10, duthost.critical_services_fully_started), "Not all critical services are fully started")
```

Comment:
I am afraid this change won't work. 

The `wait_until` function has been improved to support initial delay. Unfortunately the change is not backward compatible. The author has made a swept change. But all the open PRs still suffer from this issue.
This is the new signature of the `wait_until` function: https://github.com/Azure/sonic-mgmt/blob/master/tests/common/utilities.py#L82

For the `wait_until` to work in your local setup, can you do a rebase to the latest master branch and make a force push

Reply:
@wangxin i have reverted the changes back for the wait_until function